internal/database/external_services.go

```diff
@@ -1202,7 +1202,11 @@ FROM external_services es
                    ON es.id = essj.external_service_id
                        AND essj.state IN ('completed', 'errored', 'failed')
                        AND essj.finished_at IS NOT NULL
-WHERE ((es.namespace_user_id = %s) OR (%s AND es.namespace_user_id IS NULL))
+WHERE
+  (
+    (es.namespace_user_id = %s) OR
+    (%s AND es.namespace_user_id IS NULL AND es.namespace_org_id IS NULL)
+  )
```

Comment:
I think a correct check would be:
```suggestion
WHERE
  (
    (es.namespace_user_id = %s) OR
    (es.namespace_org_id IN (SELECT org_id FROM org_members WHERE user_id = %s)) OR 
    (%s AND es.namespace_user_id IS NULL AND es.namespace_org_id IS NULL)
  )
```

Not sure about the performance/syntax here, maybe doing a JOIN on the org_members table would do what we want as well

Reply:
But now that I think about it, this can be an improvement for a follow up PR 🤔